Forms/Main.cs

```diff
@@ -226,8 +226,9 @@ private void Main_Load(object eventSender, EventArgs eventArgs)
             this.downloadColNames.Add((int)Model.Download.DownloadCols.Duration, "Duration");
             this.downloadColNames.Add((int)Model.Download.DownloadCols.EpisodeName, "Episode Name");
             this.downloadColNames.Add((int)Model.Download.DownloadCols.Progress, "Progress");
-            this.downloadColNames.Add((int)Model.Download.DownloadCols.Status, "Status");
             this.downloadColNames.Add((int)Model.Download.DownloadCols.ProgrammeName, "Programme Name");
+            this.downloadColNames.Add((int)Model.Download.DownloadCols.SmartName, "Smart Name");
+            this.downloadColNames.Add((int)Model.Download.DownloadCols.Status, "Status");
```

Comment:
I'm probably being dense but why has the status column moved here?

Reply:
ha, this is an easy one, as really just to follow your guidance re order of these
line 224 `// NB - these are defined in alphabetical order to save sorting later`

Reply:
Gotcha, good catch.  I must have missed this when merging in the programme name changes.